main.py

```diff
@@ -2,6 +1,51 @@
+import init_django_orm # noqa:F401
 
-from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
-def main() -> QuerySet:
-    pass
+def main():
+    # Greate genres
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Drama")
+
+    # Create actors and actresses
+    Actor.objects.create(first_name="George",
+                         last_name="Klooney")
+    Actor.objects.create(first_name="Kianu",
+                         last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett",
+                         last_name="Keegan")
+    Actor.objects.create(first_name="Will",
+                         last_name="Smith")
+    Actor.objects.create(first_name="Jaden",
+                         last_name="Smith")
+    Actor.objects.create(first_name="Scarlett",
+                         last_name="Johansson")
```

Comment:
Consider using a loop with a list of tuples to create genres and actors to avoid repetition and improve code readability.